### ISSUE TITLE
Feature request--Settings menu option grayed out, can't disable checking for beta releases

### ISSUE THREAD

**Issue by mvilain** [mangerlahn/Latest#418]
When I run Latest, the Settings menu option is grayed out. I can't set up preferences. Maybe this is just a future placeholder and that's not implemented yet. Anyway, I wanted to check out what options I could set.

This is harder to implement as not everyone has the same naming convention for beta release candidates for their packages. I want an option to set to include or exclude beta releases from software to be updated. Currently I can either ignore the package entirely or just close the app when I'm done updating what I want.

**Comment by moreaki** [comment]
This looks like two separate requests.

For the disabled Preferences / Settings menu item: that part is likely already addressed in PR #557, which replaces the old storyboard-based settings window path with an explicit settings window controller and rewires the menu handling around it.

For the beta-release filtering request: I do not think that is implemented yet in any current branch.

That part is trickier because prerelease detection is not really uniform across sources. A general “include/exclude beta updates” setting would probably need source-specific logic, for example:
- Homebrew: infer prereleases from cask/formula metadata or version naming
- Sparkle: infer from appcast/version/feed conventions
- Mac App Store: likely much less reliable unless the metadata clearly distinguishes beta channels

So my read would be:
- Preferences menu issue: likely already fixed by #557
- beta filtering: still open work, and not something that can be solved cleanly with one universal rule for every source

**Comment by mvilain** [comment]
Yes, the lastest version of latest (Version 0.11 (1306)) has a working Settings menu option. It presents preferences and allows for their selection. Thanks for that.

Yeah, I figured that the ability to ignore beta packages wouldn't be straightforward because there's no standard way of naming and maintaining such things. Each application does their own thing. With MacUpdater now unsupported, latest is the only application out there that scans installed apps and reports if there are updates.

I still use MacUpdater, such as it is with it's database frozen and brew, which keeps it's own configuration of what's installed. Thanks for your effort in this application.

**Comment by moreaki** [comment]
The same here, but I'm planning on supporting this project as much as I can because I'm also stranded :). I've spent the last two days looking at dozens of issues, looking at the code trying to understand it, fixing about a dozen issues and getting a feeling on how well this software is written and if it has a future as a better replacement to MacUpdater.
